web3.md

```diff
@@ -1040,6 +1041,31 @@ Transaction Receipts
 
     rule <k> #eth_estimateGas_finalize _ => #popNetworkState ~> #sendResponse ( "error": {"code": -32000, "message":"base fee exceeds gas limit"}) ... </k>
          <statusCode> EVMC_OUT_OF_GAS </statusCode>
+```
+
+NOGAS Mode
+----------
+
+- Used for `eth_call` RPC messages
+
+```k
+    syntax Mode ::= "NOGAS"
+ // -----------------------
+    rule <k> #gas [ OP , AOP ]
+          => #if #usesMemory(OP) #then #memory [ AOP ] #else .K #fi
+          ~> #gas [ AOP ]
+         ...
+        </k>
+         <mode> MODE </mode>
+     requires MODE =/=K NOGAS
+     [priority(25)]
```

Comment:
Pretty sure we don't need this rule, if the rule below doesn't fire the one from the original semantics will.